### DISCUSSION TITLE
Error creating: pods "kube-bench-" is forbidden

### DISCUSSION THREAD

**Discussion by ErickP24** [aquasecurity/kube-bench#775] [Questions and Help]
Hi, I was using job pods but I needed to deleted for an issue in my enviroment, when I tried to deploy it again i get this error:

Error creating: pods "kube-bench-" is forbidden: PodSecurityPolicy: unable to admit pod: []

I saw https://github.com/aquasecurity/kube-bench/issues/519 this solution but it doesn't fix my issue.

**Comment by mozillazg** [comment]
You should create a PodSecurityPolicy (https://kubernetes.io/docs/concepts/policy/pod-security-policy/) for the kube-bench pods because your environment has enabled the PodSecurityPolicy feature.
